src/commands/read.ts

```diff
@@ -45,10 +45,11 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     chain,
   };
   try {
-    const res = await connect(options).read(query);
-    const formatted = format === "raw" ? res : resultsToObjects(res);
+    const res = await connect(options).read(query); // Defaults to "table"
```

Comment:
We're taking advantage of the fact that the SDK 3.1.2 release is smoothing things over for us for now.